examples/testing/schema.ts

```diff
@@ -44,4 +44,10 @@ export const lists: Lists = {
       tasks: relationship({ ref: 'Task.assignedTo', many: true }),
     },
   }),
+  Settings: singleton({
+    access: allowAll,
+    fields: {
+      siteName: text(),
+    },
+  }),
```

Comment:
`examples/testing` is meant to be about showing how to write tests for a Keystone project, not be an example for us to test Keystone itself, could you move this to `tests/sandbox` (and possibly add a singleton example if you'd like?)